layouts/fastly/helix.vcl

```diff
@@ -726,8 +733,8 @@ sub hlx_fetch_static {
       # Keep the redirect around for a short bit, to prevent thundering herd
       set beresp.cacheable = true;
       set beresp.ttl = 5s;
+      return(deliver);
     }
-    return(deliver);
```

Comment:
returning `deliver` from here will stop processing `vcl_fetch` and the ESI handling never gets executed.